tests/test_dbms.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import os
+import time
+
+import pytest
+
+import pixeltable as pxt
+from pixeltable.env import Env
+
+from .utils import reload_catalog
+
+
+class TestDbms:
```

Comment:
I put some comments below, but now I'm really confused about when/in what context this test is meant to run. If DB_CONNECT_STR is set, then that will impact the entire test suite, correct? So this test doesn't really isolate/test anything independently. And it seems to be testing general table ops, not anything specific to DBMS config.

Reply:
Removed the test

Since conftest was setting reinit_db to true it was blowing database before every run.
In the case when db_connect_str is used, ee don't want it to blow database but we can always drop tables to reset; so made that change to drop tables in db.

Now to run a specific test for cockroach all u need is set db_connect_str in config and pick which test you want to run
test_dirs.py  all tests will pass 
37 tests out of test_table.py will pass, 18 will fail
Out of 18 some((test_drop_column) are failing  because jsonb_path_query() not supported in v24 of cockroachdb. Once we get to upgrade our cluster to v25, they 'should' work.